CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+## 0.1.0
```

Comment:
Perhaps we should tag the current code in master as `1.0.0` and this would be `1.1.0`?

Also, would it make sense to adjust the Jenkins pipelines and fail ci if there are changes to the `dist` folder and no changes to the changelog?

Would version bumps (the git tag bit) be done by a jenkins pipeline after merged into master or manually?

Reply:
good call. I've just tagged `master` as `1.0.0` and changed this to `1.1.0`.

it's always fairly arbitrary when something hits the big 1.0, but you have to start somewhere and api-raml has been stable for ages.